src/main/java/org/embulk/base/restclient/JacksonServiceResponseSchema.java

```diff
@@ -18,11 +18,20 @@
 import org.embulk.spi.type.Type;
 import org.embulk.spi.type.Types;
 
+import org.embulk.base.restclient.writer.BooleanColumnWriter;
+import org.embulk.base.restclient.writer.ColumnWriter;
+import org.embulk.base.restclient.writer.DoubleColumnWriter;
+import org.embulk.base.restclient.writer.JsonColumnWriter;
+import org.embulk.base.restclient.writer.LongColumnWriter;
+import org.embulk.base.restclient.writer.SchemaWriter;
+import org.embulk.base.restclient.writer.StringColumnWriter;
+import org.embulk.base.restclient.writer.TimestampColumnWriter;
+
 import static org.embulk.spi.Exec.newConfigSource;
 
-public class SchemaWriterFactory
+public class JacksonServiceResponseSchema
```

Comment:
This class creates 1) Embulk schema and 2) handler (writer) for writing records to page objects according to the schema. For input plugins, 2) is necessary. I think that JacksonSerivceResponseSchema is not enough. How about JsonResponseHandler?